### PR TITLE
Build a SMART on FHIR Application with Oracle Autonomous Cloud Services

### PR DESCRIPTION
**Pull request by AlexandruBasarab**
### General requirements

1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

### Checklist - Refer to the QA document for the complete list

Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

### CODE REVIEW

File: smart-on-fhir/0-introduction/introduction.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Introduction
+
+## Workshop Overview
```
Comment (anooshapilli) [inline smart-on-fhir/0-introduction/introduction.md:3]: @AlexandruBasarab - Update to About this Workshop

File: smart-on-fhir/0-introduction/introduction.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Welcome to the Oracle LiveLabs Workshop: Building an ASCVD Risk Calculator! In this hands-on session, we will guide you through the process of creating a powerful application using Oracle Application Express (APEX) and FHIR APIs. By the end of this workshop, you will have a fully functional ASCVD (Atherosclerotic Cardiovascular Disease) Risk Calculator at your disposal.
+
+Estimated Time: 2 hours
```
Comment (anooshapilli) [inline smart-on-fhir/0-introduction/introduction.md:7]: @AlexandruBasarab - Rename to Estimated Workshop Time

File: smart-on-fhir/0-introduction/introduction.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+*Note: If you decide to use the below complete application, be aware that you still need to follow Sign up for Code Console lab and Task 1 from Create ASCVD Risk Calculator lab!*
+
+[ASCVD Risk Calculator](files/ascvd-risk.sql) completed application.
+
+<a href="files/fhir-client.min.js" download>SMART on FHIR</a> official JavaScript library.
+
+<a href="files/ascvd-risk.min.js" download>ASCVD Risk Calculator</a> JavaScript library.
+
+## Learn More - Useful Links
```
Comment (anooshapilli) [inline smart-on-fhir/0-introduction/introduction.md:41]: @AlexandruBasarab - Remove - useful links

File: smart-on-fhir/0-introduction/introduction.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+# Introduction
+
+## Workshop Overview
+
+Welcome to the Oracle LiveLabs Workshop: Building an ASCVD Risk Calculator! In this hands-on session, we will guide you through the process of creating a powerful application using Oracle Application Express (APEX) and FHIR APIs. By the end of this workshop, you will have a fully functional ASCVD (Atherosclerotic Cardiovascular Disease) Risk Calculator at your disposal.
+
+Estimated Time: 2 hours
+
+### During this workshop, we will cover the following steps:
```
Comment (anooshapilli) [inline smart-on-fhir/0-introduction/introduction.md:9]: @AlexandruBasarab - Please add Objectives and Prerequisites

**Review by anooshapilli — COMMENTED** [review]
@AlexandruBasarab - Please make the requested changes to the workshop.
- Remove the .ds_store files 
image

File: smart-on-fhir/0-introduction/files/ascvd-risk.min.js
Comment (ramumg) [inline smart-on-fhir/0-introduction/files/ascvd-risk.min.js:1]: image
Please follow these steps for this file.

File: smart-on-fhir/0-introduction/files/ascvd-risk.sql
Comment (ramumg) [inline smart-on-fhir/0-introduction/files/ascvd-risk.sql:1]: image
Please follow these steps for this file.

File: smart-on-fhir/0-introduction/files/fhir-client.min.js
Comment (ramumg) [inline smart-on-fhir/0-introduction/files/fhir-client.min.js:1]: image
Please follow these steps for this file.

File: smart-on-fhir/0-introduction/introduction.md (hunk 2 above)
Comment (ramumg) [reply smart-on-fhir/0-introduction/introduction.md:7]: @AlexandruBasarab, I still see it as Estimated Time instead of Workshop Time

**Review by ramumg — CHANGES_REQUESTED** [review]
@AlexandruBasarab  Please make the necessary changes that were commented so we can approve this PR.

Thank you!

**Review by denisemyrick — APPROVED** [review]

**Pull request merged**